### DISCUSSION TITLE
Assignment  Two Tasks left

### DISCUSSION THREAD

**Discussion by DoomDragoon** [Deeds101/CYBR8420-project#20] [General]
Sunday night meeting recap

These are the tasks left to do before submission Monday night please review:

Summary for:
--Ryan
--Brevin
--Corey

Contribution Reflection:
--Ryan
--Corey
--Joe
--Brevin

Group reflection (review, make contributions)

Part 2: Summarize observations related to security and installation issues

**Comment by DoomDragoon** [comment]
Link to make revisions https://github.com/Deeds101/CYBR8420-project/blob/main/Use-Misuse_Case_Submission_Doc.md

**Comment by kdherrm88** [comment]
Revised the reflection with some of what we talked about last night.
